app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ListsController < ApplicationController
+
+
+  def create
+    List.create(list_params)
+    redirect_to "/"
+  end
+
+  def show
+    @list = List.find_by(id: params[:id])
+    @user = @list.user
+  end
```

Comment:
Can you add the edit and delete functionality for these please?
@Nova840 @drucee